mingw-w64-asciidoctor-extensions/PKGBUILD

```diff
@@ -3,7 +3,7 @@
 _realname=asciidoctor-extensions
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=150.9846486
+pkgver=246.42e366e
```

Comment:
The previous [commit](https://github.com/asciidoctor/asciidoctor-extensions-lab/commit/9846486) that was used is from 2017. I find it a bit tricky to bump it so much. We can also pin it to that specific commit instead. WDYT?

Reply:
My preference is always to lag behind as little as possible without breaking things.